lib/sa.ts

```diff
@@ -1,92 +1,92 @@
-import { expect, Page } from "@playwright/test";
-import { config } from "./config";
+import { expect, Page } from '@playwright/test';
+import { config } from './config';
 
 export const navigateToSAList = async function (page: Page) {
-    await page.getByRole('link', { name: 'Application and Data Services' }).click();
```

Comment:
I'm still not sure if moving this line from here to the beforeEach in the sa.spec.ts is a good idea. Yes, it fixes the problem for your test, but not sure how it will behave in other scenarios. You need to click this button when you navigate from the console homepage. Perhaps we should use navigation through the app drawer just like the IQE test suite is using.

Reply:
It is on the `navigateToSAList` method not on the `beforeEach` method, and in case you need to click on Application and Data Services button (which is not always available depending on where you come from) you can add that step before invoking this method.

Common shared method should only contain the minimum shared set of actions.

Also I performed the regression tests run on all the tests that were invoking this method and all passed.

Reply:
Can I resolve this conversation? @mmusil